docker/dev/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:lts
+
+# Port for web server
+ENV PORT=3000
+ENV WEB_PORT=8080
+ENV REDIS_PORT=6379
+ENV REDIS_PASSWORD=redis
+ENV REDIS_DB=opentogethertubedev
+
+# Expose Port
+EXPOSE 8080
+EXPOSE 3000
+
+# Install sqlite3 and redis
+RUN apt-get update -qq
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -qq sqlite3
```

Comment:
Isn't there a flag to make apt-get non-interactive instead of using an env variable?